Makefile

```diff
@@ -1,17 +1,53 @@
+SHELL=/bin/bash
```

Comment:
These updates assume that every developer will be happy using `esy`. We should discuss this with @pveber.

Reply:
I've updated the Makefile. It no longer assumes `esy`. Instead, we assume you have ocaml and dune in your PATH somehow. How is up to you.